cleanlab/filter.py

```diff
@@ -398,6 +423,165 @@ class 0, 1, ..., K-1.
     return label_issues_mask
 
 
+def _find_label_issues_multilabel(
+    labels: list,
+    pred_probs: np.array,
+    return_indices_ranked_by: Union[None, str] = None,
+    rank_by_kwargs={},
+    filter_by: str = "prune_by_noise_rate",
+    frac_noise: float = 1.0,
+    num_to_remove_per_class: int = None,
+    min_examples_per_class=1,
+    confident_joint: Union[None, np.array] = None,
+    n_jobs: int = None,
+    verbose: bool = False,
+) -> List:
```

Comment:
Most of the mypy issues are resolved by using `np.ndarray` instead of `np.array`/`List`.

```suggestion
def _find_label_issues_multilabel(
    labels: list,
    pred_probs: np.ndarray,
    return_indices_ranked_by: Union[None, str] = None,
    rank_by_kwargs={},
    filter_by: str = "prune_by_noise_rate",
    frac_noise: float = 1.0,
    num_to_remove_per_class: int = None,
    min_examples_per_class=1,
    confident_joint: Union[None, np.ndarray] = None,
    n_jobs: int = None,
    verbose: bool = False,
) -> np.ndarray:
```